.github/workflows/plugins.yml

```diff
@@ -46,6 +46,8 @@ jobs:
       - run: yarn test:plugins:upstream
       - if: always()
         uses: ./.github/actions/testagent/logs
+      - if: always()
+        uses: ./.github/actions/testagent/results
```

Comment:
Why does this need to be added everywhere?

Reply:
But then if that's always needed, why can't it be added to the existing `log` step?

Reply:
I was initially concerned with readability and wanted it to be clear to devs that there's both test agent results and logs. I just updated the PR to just use a single step for both logs and results (with better naming).